lib/entry/LiveEntry.js

```diff
@@ -299,7 +299,8 @@ LiveEntry.prototype.entryObjectUpdated = function(entry)
         && origViewMode != kalturaTypes.KalturaViewMode.ALLOW_ALL)
     {
         this._startDVR(entry);
-        this._removeClipTo();
+        const persistePlaylistPromise = this._removeClipTo();
```

Comment:
better if you do this._removeClipTo().then ( ()=> this._updateIsLiveUser(entry, true) )